### DISCUSSION TITLE
Unable to decompress a bzip2 stream

### DISCUSSION THREAD

**Discussion by Tsury** [101arrowz/fflate#183] [Q&A] [answered]
Either it's impossible or I'm just doing something wrong, but I'm getting an `invalid distance` error when trying to decompress a bzip2 stream.

Here's a [reproduction](https://replit.com/@Tsury/bzip2test#index.js) of my issue - I'd appreciate a helping hand, thanks!

**Comment by 101arrowz** [comment, marked as answer]
Sorry; bzip2 is not natively supported by `fflate`. I see now that the note in the README is confusing; I'll update it to make it more clear.

**Reply by Tsury** [reply]
Thanks. Out of desperation - do you happen to know of a way to decompress a stream of BZip2 on the browser? Really having issues achieving this, was hoping fflate would be my savior, but seems like I have to keep searching...

**Reply by 101arrowz** [reply]
I'd try https://github.com/SheetJS/bz2 - it's not super well maintained but given that bzip2 is as old as it is, that probably doesn't matter.

**Reply by Tsury** [reply]
Will do, thanks again.
